OmniKit/PumpManager/PodInsulinMeasurements.swift

```diff
@@ -15,16 +15,16 @@ public struct PodInsulinMeasurements: RawRepresentable, Equatable {
     public let delivered: Double
     public let reservoirVolume: Double?
     
-    public init(statusResponse: StatusResponse, validTime: Date, setupUnitsDelivered: Double?) {
+    public init(insulinDelivered: Double, reservoirLevel: Double?, setupUnitsDelivered: Double?, validTime: Date) {
         self.validTime = validTime
-        self.reservoirVolume = statusResponse.reservoirLevel
+        self.reservoirVolume = reservoirLevel
```

Comment:
[nit] reservoirLevel seems to be a better name than reservoirVolume (which could be interpreted as meaning the entire volume of the reservoir, regardless of contents).  Maybe we can update this while we're changing this code.